packages/graphback-runtime-knex/src/KnexDBDataProvider.ts

```diff
@@ -81,6 +81,15 @@ export class KnexDBDataProvider<Type = any, GraphbackContext = any> implements G
     throw new NoDataError(`Cannot find all results for ${this.tableName}`);
   }
 
+  public async findMore(offset: number, limit: number): Promise<Type[]> {
```

Comment:
Changed my mind. If we keep those args optional we can extend existing function without breaking API.